.github/workflows/autopep8.yml

```diff
@@ -9,20 +9,21 @@ on:
   pull_request:
     branches: [ "main", "feature/autopep8" ]
```

Comment:
The branches needs to be changed to [ "main", "feature/*", "*" ]

Reply:
done